locales/en-US/experiments.ftl

```diff
@@ -143,7 +143,7 @@ snoozetabsContributors5Title = Firefox QA
 snoozetabsContributors6Title = Softvision QA
 snoozetabsContributors7Title = Softvision QA
 tabcenterDescription = What would it be like to move tabs from the top of the browser to the side? We wanted to find out!
-tabcenterIntroduction = <p>Take your tabs from in the way to tucked away. Tab Center moves your tabs to the side of your browser window so they're out of sight when you don't need them, and easy to expand and grab when you do.</p>
+tabcenterIntroductionWarning = <div class="warning"><strong>Firefox is changing fast. Tab Center may have bugs in Firefox 55 and will be disabled completely in Firefox 56+. </strong></div> <p>Take your tabs from in the way to tucked away. Tab Center moves your tabs to the side of your browser window so they're out of sight when you don't need them, and easy to expand and grab when you do.</p>
```

Comment:
Besides the markup, it would be asking people to retranslate the whole sentence, when only the first bit is added. Ideally it should only be

```
tabcenterIntroductionWarning = Firefox is changing fast. Tab Center may have bugs in Firefox 55 and will be disabled completely in Firefox 56+. 
```